.github/actions/build-test/windows/action.yml

```diff
@@ -0,0 +1,168 @@
+name: Build & Test - Windows
+
+inputs:
+  GH_TOKEN:
+    required: true
+  BOOST_VERSION:
+    required: true
+  THRIFT_VERSION:
+    required: true
+  BUILD_TYPE:
+    required: true
+  SHARED_LIBS_TOGGLE:
+    required: true
+  OPENSSL_TOGGLE:
+    required: true
+  ARCH_CHOCO_OPTIONS:
+    required: true
+  ARCH_ADDRESS_MODEL:
+    required: true
+  ARCH_CMAKE:
+    required: true
+  INSTALL_BOOST:
+    required: true
+  INSTALL_THRIFT:
+    required: true
+  BOOST_URL:
+    required: true
+  BOOST_ARCHIVE_NAME:
+    required: true
+  BOOST_FOLDER_NAME:
+    required: true
+  BOOST_INCLUDE_FOLDER:
+    required: true
+  RUN_TESTS:
+    required: true
+  HAZELCAST_ENTERPRISE_KEY:
+    required: true
+  AWS_ACCESS_KEY_ID:
+    required: true
+  AWS_SECRET_ACCESS_KEY:
+    required: true
+  HZ_TEST_AWS_INSTANCE_PRIVATE_IP:
+    required: true
+
+env:
+  # Not possible to set this as a default
+  # https://github.com/orgs/community/discussions/46670
+  shell: pwsh
+
+runs:
+  using: composite
+  steps:
+    - name: Read Config
+      shell: bash
+      run: cat .github/config.env >> $GITHUB_ENV
```

Comment:
why not use pwsh?
something like `echo "{name}={value}" >> $Env:GITHUB_ENV`

Reply:
> why not use pwsh? something like `echo "{name}={value}" >> $Env:GITHUB_ENV`

Consistency with the other places this is done, as it works with `bash` on Windows (somehow).

Reply:
ok. not an issue. just little odd as would have preferred to stick with wins commands

I tried `sh` in my BAT console and I get 
![image](https://github.com/user-attachments/assets/c54b43dc-96b8-4f33-8376-1e69479f9451)

Looks like installed with Git
![image](https://github.com/user-attachments/assets/1841897a-9c64-4fee-a210-979348f9df7f)

Also I know `bash` actually runs in WSL in Windows for me But git also provides `bash.exe`